packages/twenty-website/src/app/user-guide/constants/UserGuideIndex.ts

```diff
@@ -0,0 +1,29 @@
+export type IndexSubtopic = {
+  title: string;
+  url: string;
+};
+
+export type IndexHeading = {
+  [heading: string]: IndexSubtopic[];
+};
+
+export const UserGuideIndex = {
```

Comment:
Let's try to keep the previous script that iterates over markdown files?

Reply:
But then this design can't be followed as earlier way of extracting was using components with child items and here we are using state on selected item so shall I proceed?

Reply:
I mean is then figma design can't be replicated. I followed this layout as a thought that there won't be much change in the index